src/bouwdossiers/batch.py

```diff
@@ -595,6 +604,7 @@ def add_bag_ids_to_pre_wabo():
 SET openbareruimte_id = opr.landelijk_id
 FROM bag_openbareruimte opr
 WHERE ba.straat = opr.naam
+AND opr.id LIKE '0363%'
 AND (ba.openbareruimte_id IS NULL OR ba.openbareruimte_id = '')
         """)
     log.info("Finished adding openbare ruimtes")
```

Comment:
Ik mis hier ook nog een unit test voor

Reply:
Voor deze hele stap zijn nog geen unit tests. Dat gaat erg veel tijd kosten om voor de unittest (een subset van) de bag te importeren. Dit is lokaal wel getest.